### ISSUE TITLE
Cannot submit for review

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-app#405]
As a guest, I cannot submit my event to "Send for review".

Reproduced in production using Safari. 

The event already existed in draft mode. 

![Image](https://github.com/culturecreates/footlight-app/assets/419491/593f8701-9603-44b9-94fd-c1d7f1705a24)

**Comment by saumier** [comment]
Gregory to add steps to reproduce

**Comment by saumier** [comment]
![Image](https://github.com/culturecreates/footlight-app/assets/419491/4ba60ca7-14b4-4d35-a98f-b9251aac2df7)

**Comment by saumier** [comment]
@saumier @troughc Try to find repro steps.

**Comment by troughc** [comment]
@saumier @AbhishekPAnil I tried two different approaches after logging is as a guest user. One did not reproduce the error, the other one did. I did this at 13:40-ish

This does NOT reproduce the error:
1. duplicate event.
2. rename the event and then send for review. The event is sent for review. I get the success message and am back on the event listing page.
3. I reenter the event - it is read only as expected


SUCCESSFULLY reproduced this issue doing the folloiwng steps:
1. duplicate original event that cannot be published (above)
2. save as draft
3. reopen my new duplicated  event (i then modified the title name to add test)
4. Try to send for review. I get this error:

![Image](https://github.com/culturecreates/footlight-app/assets/64489312/c3eaa8a2-96cf-4fdb-a58f-97d4318090df)

5. I leave and come back. I try to save as draft. I get this error:


![Image](https://github.com/culturecreates/footlight-app/assets/64489312/2a27e4c7-a673-4879-bdef-b51d192ffe93)

6. I leave and come back. I am scrolled to this position in the UI:


![Image](https://github.com/culturecreates/footlight-app/assets/64489312/eede5fc8-7bc4-4d3c-a3ad-e8eb4d578613)

7. I open the developer tools and see this:


![Image](https://github.com/culturecreates/footlight-app/assets/64489312/7ec911cc-51e0-4c8a-a53e-c3a440f6a88c)

8. I cleared the developer tools and click send for review. I get the error and these results:



![Image](https://github.com/culturecreates/footlight-app/assets/64489312/6baae4f5-44cb-4072-820f-4bc6ec7749b5)



![Image](https://github.com/culturecreates/footlight-app/assets/64489312/ff09e4ea-f0f5-4fba-8a39-3d332c7654a8)

**Comment by troughc** [comment]
1. I duplicated the event one more time (appended TEST 3 to event name) and as soon as I did, this is the message I got:


![Image](https://github.com/culturecreates/footlight-app/assets/64489312/ec920486-0292-4534-a322-74e290503b34)

2. Rename and save event - this works
3. Reopen event and send to review - this worked. The only difference is that I modified the name before saving instead of after saving.

**Comment by troughc** [comment]
I am stopping at 2:12 EST

**Comment by saumier** [comment]
@troughc @AbhishekPAnil @dev-aravind This issue has a custom fix that covers the dynamic field "Pressoir..." for Events in Tout Culture.  This should unblock guests and contributors unable to save or send to review.  However it does not fix the general problem of admin only dynamic fields. I am opening another issue to address the general problem of guests editing events with admin only properties. The "reproduction steps" will need to be redone with a new admin-only field that needs to be setup in staging. The dev team will need to have a meeting to discuss how to address.

**Comment by saumier** [comment]
@troughc The workaround for events created before today is as follows: An admin edits the event in question, adds "pressoir", saves the event, then removed "pressoir" and saves the event again.  So if a guest gets stuck again with an old event created before today with the message "Something went wrong! Cannot read property "taxonomyId"..." then this workaround can help them.
